spec/unit/webrtc/callFeed.spec.ts

```diff
@@ -32,30 +30,60 @@ describe("CallFeed", () => {
         client.stop();
     });
 
-    it("should handle stream replacement", () => {
-        const feedNewStreamCallback = jest.fn();
-        const feed = new CallFeed({
-            client,
-            roomId,
-            userId: "user1",
-            // @ts-ignore Mock
-            stream: new MockMediaStream("stream1"),
-            id: "id",
-            purpose: SDPStreamMetadataPurpose.Usermedia,
-            audioMuted: false,
-            videoMuted: false,
+    describe("muting", () => {
+        let feed: CallFeed;
+
+        beforeEach(() => {
+            feed = new CallFeed({
+                client,
+                roomId: "room1",
+                userId: "user1",
+                // @ts-ignore Mock
+                stream: new MockMediaStream("stream1"),
+                purpose: SDPStreamMetadataPurpose.Usermedia,
+                audioMuted: false,
+                videoMuted: false,
+            });
+        });
+
+        describe("should by muted by default", () => {
+            it("audio", () => {
```

Comment:
the describe/it usage is backwards here: "describe" is used to tell the reader the general area, and "it" starts with a "should ..." sentence. 

Suggestion would be to name it like:
```
muting by default
  should mute audio by default
  should mute video by default
```